src/DotNetWorker/OutputBindings/MethodReturnOutputBindingsInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.OutputBindings
+{
+    public class MethodReturnOutputBindingsInfo : OutputBindingsInfo
+    {
+        private const string ReturnBindingName = "$return";
```

Comment:
Should this constant be defined more broadly? Isn't this used in other places?

Reply:
Yes I had defined it in `OutputBindingsConstants` but forgot to use here. sound like this wouldn't be required anyway.